scripts/dns-delay/install-dns-delay.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+tc qdisc del dev "$(route | grep '^default' | grep -o '[^ ]*$')" root 2>/dev/null || true
+tc qdisc add dev "$(route | grep '^default' | grep -o '[^ ]*$')" root handle 0: mq || true
+tc qdisc del dev eth0 root 2>/dev/null || true
```

Comment:
Do we expect eth0 to be up by the time this script runs? If not, we might need "while ! ip link | grep "eth0:" > /dev/null; do sleep 1; done"

I am not an expert on this.. just comparing the workaround in the blog and here :)

Reply:
Most likely eth0 will be up by the time netd starts - but no harm to add that :) Will do.

Reply:
Suggestion applied.